app/src/main/java/org/oppia/android/app/databinding/MarginBindingAdapters.java

```diff
@@ -1,35 +1,61 @@
 package org.oppia.android.app.databinding;
 
+import android.annotation.SuppressLint;
 import android.view.View;
 import android.view.ViewGroup.MarginLayoutParams;
+
 import androidx.annotation.NonNull;
+import androidx.core.view.ViewCompat;
 import androidx.databinding.BindingAdapter;
 
-/** Holds all custom binding adapters that set margin values. */
+/**
+ * Holds all custom binding adapters that set margin values.
+ */
 public final class MarginBindingAdapters {
 
-  /** Used to set a margin-start for views. */
+  @SuppressLint("WrongConstant")
+  private static boolean isRtlLayout(View view) {
+    return view.getLayoutDirection() == ViewCompat.LAYOUT_DIRECTION_RTL;
+  }
+
+  /**
+   * Used to set a margin-start for views.
```

Comment:
Here & elsewhere: method Javadocs should be a singular third-person verb or verb phrase. To adjust this one to match:

```java
/** Sets the start margin for a view, accounting for RTL scenarios. */
```

(Plus the additional RTL bit that's being added here).

Ditto elsewhere.

Reply:
Done.